pkg/kp/pcstore/store.go

```diff
@@ -0,0 +1,13 @@
+package pcstore
+
+import (
+	"github.com/square/p2/pkg/pc/fields"
+	"github.com/square/p2/pkg/types"
+)
+
+const podClusterTree string = "pod_clusters"
+
+type Store interface {
+	Create(podId types.PodID, availabilityZone fields.AvailabilityZone, clusterName fields.ClusterName) (fields.PodCluster, error)
```

Comment:
I need Annotations here

I deliberately didn't put pod selector here because it will be constructed from the other types. Maybe someday we'll want more flexbility